smart-contracts/contracts/CryptoTrees.sol

```diff
@@ -72,7 +72,7 @@ contract CryptoTrees is BasicERC721, Ownable {
         //      ~40% chance the inviter mints a new token
         //      ~10% chance the invitee mints a new token
         //      ~50% chance nothing happens
-        uint randomNumber = uint(block.blockhash(block.number-1)) % 100 + 1;
+        uint randomNumber = uint(blockhash(block.number-1)) % 100 + 1;
```

Comment:
This fails on my side. Is there something I need to do (update my solc version?) if I want to have that syntax available?